Sources/PointFree/About.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Prelude
+import Styleguide
+
+let aboutResponse: Middleware<StatusLineOpen, ResponseEnded, Prelude.Unit, Data> =
+  writeStatus(.ok)
+    >-> respond(aboutView)
+
+private let aboutView = View<Prelude.Unit> { _ in
+  document([
+    html([
+      head([
+        style(renderedNormalizeCss),
+        style(styleguide),
```

Comment:
need to come up with nice, reusable, composable layouts for abstracting away this stuff. I think there's a few directions we can go, and not sure of the best one...